tbtc-ts/test/data/bitcoin.ts

```diff
@@ -43,3 +45,230 @@ export const testnetUTXO: UnspentTransactionOutput & RawTransaction = {
   value: 3933200,
   ...testnetTransaction,
 }
+
+/**
+ * Private key of the wallet on testnet.
+ */
+export const testnetWalletPrivateKey =
+  "cRk1zdau3jp2X3XsrRKDdviYLuC32fHfyU186wLBEbZWx4uQWW3v"
+
+/**
+ * Address corresponding to testnetWalletPrivateKey.
+ */
+export const testnetWalletAddress = "tb1q3k6sadfqv04fmx9naty3fzdfpaecnphkfm3cf3"
+
+/**
+ * Address generated from deposit script hash during deposit creation
+ */
+export const testnetDepositScripthashAddress =
+  "2Mxy76sc1qAxiJ1fXMXDXqHvVcPLh6Lf12C"
+
+/**
+ * Address generated from deposit witness script hash during deposit creation
+ */
+export const testnetDepositWitnessScripthashAddress =
+  "tb1qs63s8nwjut4tr5t8nudgzwp4m3dpkefjzpmumn90pruce0cye2tq2jkq0y"
+
+export const NO_PREVIOUS_SWEEP = {
+  transactionHash: "",
+  outputIndex: 0,
+  value: 0,
+}
+
+/**
+ * Represents data for tests of assembling sweep transactions.
+ */
+export interface SweepDepositsTestData {
```

Comment:
Just `SweepTestData`?

Reply:
Done in https://github.com/keep-network/tbtc-v2/pull/96/commits/3822ba1ad8310a8f5c0a2d2d98e9ed548a1d5262.